plugin/init.lua

```diff
@@ -211,7 +238,14 @@ end
 ---@param name string
 ---@param type string
 function pub.load_state(name, type)
-	return load_json(get_file_path(name, type))
+	wezterm.emit("resurrect.load_state.start", name, type)
+	local json = load_json(get_file_path(name, type))
+	if not json then
+		wezterm.emit("resurrect.error", "Invalid json: " .. name, type)
```

Comment:
Maybe this should be `"Invalid json: " .. get_file_path(name, type)

Reply:
Will update.